docs/release_notes/v-1.5.md

```diff
@@ -0,0 +1,67 @@
+# Airbyte 1.5.0
+
+Valentine's Day let you down? That's OK. You'll love this. Airbyte 1.5.0 was released on February 19, 2025. We’re excited to share new improvements and changes to the Airbyte platform.
+
+## 🚀 Self-Managed Enterprise Changes
```

Comment:
I suggest starting with the platform changes (for all) before the enterprise changes :)